dexter/src/main/java/com/karumi/dexter/AndroidPermissionService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 Karumi.
+ */
+
+package com.karumi.dexter;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Wrapper class for all the static calls to the Android permission system
+ */
+class AndroidPermissionService {
```

Comment:
It makes sense to have this class with a ``default`` visibility modifier and the methods with ``public``?